treemap.swt/src/main/java/de/engehausen/treemap/swt/TreeMap.java

```diff
@@ -228,6 +235,17 @@ public ITreeModel<N> getCurrentTreeModel() {
 		return model;
 	}
 
+	/**
+	 * Defines whether the mouse cursor should be changed to {@link SWT#CURSOR_WAIT} during recalculation.
+	 *
+	 * @param changeCursorOnRecalculate State to be set.
+	 *
+	 * @author Mario Marinato
+	 */
+	public void setChangeCursorOnRecalculate( boolean changeCursorOnRecalculate ) {

```

Comment:
The source code formatting I am using does not use whitespaces before or after the parentheses, and I use final when possible (reasoning: http://archive.today/ZIwdk) - so this should be `public void setChangeCursorOnRecalculate(final boolean changeCursorOnRecalculate) {`

Since there is a setter, an equivalent getter should be provided; as boolean, that would typically be `public boolean isChangeCursorOnRecalculate() { return changeCursorOnRecalculate; }` (with JavaDoc, of course).

Reply:
As we're going to change the constructor to add the parameter, I'd drop this method.